### PR TITLE
Add examples to scipy.ndimage.filters

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: scipy/ndimage/filters.py [hunk 1]
```diff
@@ -337,6 +337,14 @@ def prewitt(input, axis=-1, output=None, mode="reflect", cval=0.0):
     %(output)s
     %(mode)s
     %(cval)s
+
+    Examples
+    --------
+    >>> import scipy.misc
```
Comment (rgommers) [inline scipy/ndimage/filters.py]: Import style: ``from scipy import ndimage, misc``

File: scipy/ndimage/filters.py [hunk 2]
```diff
@@ -337,6 +337,14 @@ def prewitt(input, axis=-1, output=None, mode="reflect", cval=0.0):
...
+    >>> ascent = scipy.misc.ascent()
+    >>> from scipy.ndimage import prewitt
+    >>> result = prewitt(ascent)
+    >>> scipy.misc.imshow(result)
```
Comment (rgommers) [inline scipy/ndimage/filters.py]: at the top of the example: ``import matplotlib.pyplot as plt``

and then here: ``plt.imshow(result)``

**Comment by rgommers** [comment]
And a minor thing for next commits: please use the prefixes given in http://docs.scipy.org/doc/numpy-dev/dev/gitwash/development_workflow.html#writing-the-commit-message

**Comment by rgommers** [comment]
Overall +1, nice to have some examples for ``ndimage``.

**Comment by MartinThoma** [comment]
Thank you for pointing me to the commit prefixes; I adjusted my commit message (and changed the other things you mentioned).

**Comment by rgommers** [comment]
Code looks good now, but the example is a pretty bad green: https://circle-artifacts.com/gh/scipy/scipy/145/artifacts/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.ndimage.prewitt.html#scipy.ndimage.prewitt

How about adding ``plt.gray()`` before ``plt.imshow`` to show the filtered result in grayscale? Another colormap is fine with me too, but the current default is not great.

**Comment by MartinThoma** [comment]
@rgommers Thank you for the hint; I've added it: https://circle-artifacts.com/gh/scipy/scipy/146/artifacts/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.ndimage.prewitt.html#scipy.ndimage.prewitt

**Comment by rgommers** [comment]
That looks much better. One last issue - the second figure for ``gaussian_laplace`` doesn't show up: https://circle-artifacts.com/gh/scipy/scipy/146/artifacts/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.ndimage.gaussian_laplace.html#scipy.ndimage.gaussian_laplace

If you insert a ``plt.figure()`` above the last 2 lines it probably will. Then I think you'll get 2 figures below each other. Better would be to have them next to each other - this can be done by creating 2 axes in a figure, like:

    fig = plt.figure()
    plt.gray()
    ax1 = fig.add_subplot(121)
    ax1.imshow(...)
    ax2 = fig.add_subplot(122)
    ax2.imshow(...)

**Comment by MartinThoma** [comment]
Yes, that looks nicer: https://circle-artifacts.com/gh/scipy/scipy/152/artifacts/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.ndimage.gaussian_laplace.html#scipy.ndimage.gaussian_laplace

**Comment by rgommers** [comment]
Much better. In it goes. Thanks @MartinThoma!

**Pull request merged**